src/providers/mbed_provider/generate_random.rs

```diff
@@ -0,0 +1,27 @@
+use super::MbedProvider;
+use crate::authenticators::ApplicationName;
+use parsec_interface::operations::psa_generate_random;
+use parsec_interface::requests::{ResponseStatus, Result};
+use psa_crypto::operations::other::generate_random;
+
+impl MbedProvider {
+    pub(super) fn psa_generate_random_internal(
+        &self,
+        _app_name: ApplicationName,
+        op: psa_generate_random::Operation,
+    ) -> Result<psa_generate_random::Result> {
```

Comment:
Now that I look at it, you could just put this one straight in the `psa_generate_random` method, there isn't that much code to warrant an `_internal` one, but feel free to keep it here

Reply:
I think I'll leave this -- it looks like most of the operations have an `*_internal` method. If you have strong opinions here feel free to reply and I'll change. 😄